tests/unit/electionguard_gui/test_decryption_dto.py

```diff
@@ -0,0 +1,100 @@
+from datetime import datetime
+
+from unittest.mock import MagicMock, patch
+
+from electionguard_gui.models.decryption_dto import DecryptionDto
+from tests.base_test_case import BaseTestCase
+
+
+class TestDecryptionDto(BaseTestCase):
+    """Test the DecryptionDto class"""
+
+    def test_get_status_with_no_guardians(self) -> None:
+        # ARRANGE
+        decryption_dto = DecryptionDto(
+            {
+                "guardians_joined": [],
+                "guardians": 2,
+            }
+        )
+
+        # ACT
+        status = decryption_dto.get_status()
+
+        # ASSERT
+        self.assertEqual(status, "waiting for all guardians to join")
+
+    def test_get_status_with_all_guardians_joined_but_not_completed(self) -> None:
+        # ARRANGE
+        decryption_dto = DecryptionDto(
+            {"guardians_joined": ["g1", "g2"], "guardians": 2, "completed_at_utc": None}
+        )
+
+        # ACT
+        status = decryption_dto.get_status()
+
+        # ASSERT
+        self.assertEqual(status, "performing decryption")
+
+    def test_get_status_with_all_guardians_joined_and_completed(self) -> None:
+        # ARRANGE
+        decryption_dto = DecryptionDto(
+            {
+                "guardians_joined": ["g1"],
+                "guardians": 1,
+                "completed_at": datetime.utcnow(),
+            }
+        )
+
+        # ACT
+        status = decryption_dto.get_status()
+
+        # ASSERT
+        self.assertEqual(status, "decryption complete")
+
+    @patch("electionguard_gui.services.authorization_service.AuthorizationService")
```

Comment:
This demonstrates how to instantiate a mock object that can be passed to another method as a parameter